notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-095_settling_big_vibrations.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<p>I guess that the arrow indicates the settle time that has been determined. I suggest to indicate it in the legend of the plot</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>